Input/input.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="../../Utility CSS/utility.css" />
+    <link rel="stylesheet" href="./input.css" />
+
+    <link rel="icon" href="../../img/logo.png" type="image/x-icon" />
+    <title>Card</title>
+  </head>
+  <body>
+    <header>
+      <nav class="container-flex flex-sb flex-wr">
+        <div class="header__nav">
+          <ul class="container-flex">
+            <li>
+              <a
+                href="../../index.html"
+                class="font-l anchor-style nav__icon--position nav__item--hover"
+                >CLI
+                <span class="nav__subtitle--position">COMPONENT LIBRARY APP</span>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <div class="container">
+      <div class="sidebar container-flex">
+        <ul class="font-xl">
+          <li>
+              <span class="sidebar-title"> Components </span>
+          </li>
+          <li>
+              <a href="../Avatar/avatar.html" class="sidebar-menu" rel="noopener noreferrer">
+              Avatar
+              </a>
+          </li>
+          <li>
+              <a href="../Alert/alert.html" class="sidebar-menu" rel="noopener noreferrer">
+              Alert
+              </a>
+          </li>
+          <li>
+              <a href="../Badge/badge.html" class="sidebar-menu" rel="noopener noreferrer">
+              Badge
+              </a>
+          </li>
+          <li>
+              <a href="../Button/button.html" class="sidebar-menu" rel="noopener noreferrer">
+              Button
+              </a>
+          </li>
+          <li>
+              <a href="../Card/card.html" class="sidebar-menu" rel="noopener noreferrer">
+              Card
+              </a>
+          </li>
+          <li>
```

Comment:
what's the use of rel tag here?

Reply:
it tells the browser to open the link in new tab.